tasks/skylab/StarAlign.wdl

```diff
@@ -65,3 +65,69 @@ task StarAlignBamSingleEnd {
   }
 
 }
+
+task StarAlignFastqPairedEnd {
+  input {
+    File fastq1
+    File fastq2
+    File tar_star_reference
+
+    # runtime values
+    String docker = "quay.io/humancellatlas/secondary-analysis-star:v2.7.8a"
```

Comment:
We decided not to upgrade to v2.7.9a?

Reply:
We did, not sure how it got reverted. Updated back to 2.7.9a now.